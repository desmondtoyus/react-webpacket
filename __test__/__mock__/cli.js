const inquirer = require('inquirer');
 const {QUESTIONS} =require('../../contents/module')
 inquirer.prompt(QUESTIONS)
 .then((answers) =>{
  return`${answers['projectChoice']}`;
});
