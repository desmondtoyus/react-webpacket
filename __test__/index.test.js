'use strict';

jest.mock('fs');

describe('listFilesInDirectorySync', () => {
  const MOCK_FILE_INFO = {
    '/path/to/file1.js': 'console.log("file1 contents");',
    '/path/to/file2.txt': 'file2 contents',
  };

  beforeEach(() => {
    // Set up some mocked out file info before each test
    require('./__mock__/fs').__setMockFiles(MOCK_FILE_INFO);
  });

  test('includes all files in the directory in the summary', () => {
    const FileSummarizer = require('../index');
    const fileSummary = FileSummarizer.summarizeFilesInDirectorySync(
      './',
    );

    expect(fileSummary.length).toBe(2);
  });
});