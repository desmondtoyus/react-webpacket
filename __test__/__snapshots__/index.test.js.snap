// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI TEST should return an object 1`] = `
"[32m?[39m [1mSelect react-redux-webpack project. [22m[0m [0m[2m(Use arrow keys)[22m
[36m‚ùØ react-redux-webpack[39m [22D[22C[2K[1A[2K[G[32m?[39m [1mSelect react-redux-webpack project. [22m[0m [0m[36mreact-redux-webpack[39m[58D[58C
[32m?[39m [1mViews (Separated by a space, Enter to Ignore):[22m[0m [0m[49D[49C[2K[G[32m?[39m [1mViews (Separated by a space, Enter to Ignore):[22m[0m [0mh[50D[50C[2K[G[32m?[39m [1mViews (Separated by a space, Enter to Ignore):[22m[0m [0mho[51D[51C[2K[G[32m?[39m [1mViews (Separated by a space, Enter to Ignore):[22m[0m [0mhom[52D[52C[2K[G[32m?[39m [1mViews (Separated by a space, Enter to Ignore):[22m[0m [0mhome[53D[53C[2K[G[32m?[39m [1mViews (Separated by a space, Enter to Ignore):[22m[0m [0m[36mhome[39m[53D[53C
[32m?[39m [1mRedux Reducers (Separated by a space, Enter to Ignore):[22m[0m [0m[58D[58C[2K[G[32m?[39m [1mRedux Reducers (Separated by a space, Enter to Ignore):[22m[0m [0ma[59D[59C[2K[G[32m?[39m [1mRedux Reducers (Separated by a space, Enter to Ignore):[22m[0m [0mau[60D[60C[2K[G[32m?[39m [1mRedux Reducers (Separated by a space, Enter to Ignore):[22m[0m [0maut[61D[61C[2K[G[32m?[39m [1mRedux Reducers (Separated by a space, Enter to Ignore):[22m[0m [0mauth[62D[62C[2K[G[32m?[39m [1mRedux Reducers (Separated by a space, Enter to Ignore):[22m[0m [0m[36mauth[39m[62D[62C
"
`;
